src/schemas/researchDomain.ts

```diff
@@ -0,0 +1,29 @@
+import gql from 'graphql-tag';
+
+export const typeDefs = gql`
+  extend type Query {
+    "Get all the QuestionTypes"
+    researchDomains: [ResearchDomain]
+  }
+
+  "An aread of research (e.g. Electrical Engineering, Cellular biology, etc.)"
+  type ResearchDomain {
+    "The unique identifer for the Object"
+    id: Int
+    "The user who created the Object"
+    createdById: Int
+    "The timestamp when the Object was created"
+    created: String
+    "The user who last modified the Object"
+    modifiedById: Int
+    "The timestamp when the Object was last modifed"
+    modified: String
+    "Errors associated with the Object"
+    errors: [String!]
+
+    "The name of the domain"
+    name: String!
```

Comment:
The ticket has `uri` included as a field, but I don't see it here. Just checking whether it's necessary.

Reply:
Ah thanks good catch